draft-ietf-rats-corim.md

```diff
@@ -1066,6 +1072,328 @@ The following describes each member of the `concise-bom-tag` map.
+whether a candidate CoMID tag matches the Accepted Claims Set.
+If a tag does not match the Accepted Claims Set then it is silently ignored
+for this pass, the Verifier continues to iterate over the remaining candidate
+tags.
+
+The Verifier iterates over the Reference Values in the CoMID tag.
+If the Accepted Claims Set does not contain an entry with the same `environment-map`
+as the Reference Value then the tag does not match.
+Comparison of `environment-map` is performed using a binary comparison.
+A Verifier SHOULD convert `environment-map` to the canonical format before
+performing the binary comparison.
+
+The Verifer locates the `measurement-values-map` corresponding to the
+`environment-map` in the Reference Value and the Accepted Claims Set.
+The Verifier enumerates over the key-value pairs in the Reference Value
+`measurement-values-map`.
+If the Accepted Claims Set `measurement-values-map` does not have an entry
+with the same key as the Reference Value `measurement-values-map`,
+or if the entries do not match, then the tag does not match.
+
+The algorithm used to match the `measurement-values-map` entries together
+depends on whether the reference value is tagged,
+and on the `measurement-values-map` key which identifies the entry.
+
+If the Reference Value `measurement-values-map` value starts with a CBOR tag
+then the verifier MUST use the algorithm associated with that tag to match
+the entries.
+Handling for initial tags is described in sub-sections below.
+If the verifier does not recognise the tag value then the tag does not match.
+
+If the Reference Value is not tagged and the measurement-value-map key is a
+special value described in the sub-sections below,
+then the algorithm appropriate to that key is used to match the entries.
+
+If the Reference Value is not tagged, and the `measurement-values-map` key
+is not a special value described below,
+then the entries are compared using binary comparison of their CBOR values.
+
+Note that while specifications may extend the matching semantics using tags,
+there is no way to extend the matching semantics of special key values.
+Any new keys requiring special handling must have an appropriate tag in the
+reference value.
+
+#### Comparing raw-value entries
+
+[^issue]: Content missing. Tracked at https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/71
+
+#### Comparing svn entries
+
+[^issue]: Content missing. Tracked at https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/71
+
+#### Comparing digests entries
+
+[^issue]: Content missing. Tracked at https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/71
+
+#### Defining handling for new tags
+
+[^issue]: Content missing. Tracked at https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/71
+
+### Adding CoMID Endorsed Values to the Accepted Claims Set
+
+If a CoMID tag matches the Accepted Claims Set then the verifier must attempt
```

Comment:
Not sure what we meant by _"must attempt"_.  It this a best-effort operation?  (I don't think so.)

Reply:
(as suggested above) matching should be described as the Claim level of granularity. For a CoMID tag to "match" Endorsed values, the tag must contain triples that contain Endorsed Values. The Environment that is the subject of the triple (aka the name portion of the 'name-value' statement) must equal the name portion of a 'name-value' statement found in ACS. This is closer to what it means to "match" something. The measurements in the 'value' portion of the Endorsed Values claim is copied into the ACS at the place where the name match occurred. 
Note: If there are Reference Value claims that didn't match Evidence then the node in the ACS where the Endorsement claim would have been copied into will be removed including any claims that might have shown up in advance, be it from processing Endorsements or from processing Attester Evidence claims are are the superset of the Reference Values claims.  
By describing the acceptance and rejection criteria, the explanation can avoid having to be careful about building up state of the ACS over multiple passes and keeping track to apply the right pass at the right time.